tests/outputs/cv/scopatz.tex

```diff
@@ -240,8 +240,8 @@
 \ressubheadingitems{SymPy 1.1 Release Support}
     { \$3,000.00}
     {NumFOCUS Small Development Grants }
-    {May 2030 -
-      Dec 2030}
+    {present -
```

Comment:
this doesn't look right.  I think there should be a date here.  When tests fail it either means a bug in the code to fix, or the test to fix.....only fix the test if you are sure there is not code bug....

Reply:
That is right. That was the grant I removed the dates from. This way when I run grant_lister, it looks for the dates in the linking proposal.